tests/cli/test_cli.py

```diff
@@ -1470,6 +1486,22 @@ def test_run2(ctx):
         [cl, 'run', 'foo:%s' % dir3, 'foo/bar:%s' % dir1, 'cat foo/bar/f1'], expected_exit_code=1
     )
 
+    # Test that cpu_usage and memory_limit are properly populated
+    uuid = _run_command([cl, 'run', "bash -c 'for i in {1..30}; do sleep 1; done'"])
+    wait_until_running(uuid)
```

Comment:
I think we can replace this function with just using the existing function, `wait_until_state`.